doc/nrf/known_issues.rst

```diff
@@ -2010,6 +2010,16 @@ NCSDK-12483: Missing debug symbols
 
   **Workaround:** Add the following text in the application :file:`CMakeLists.txt` file: ``zephyr_link_libraries(-Wl,--undefined=my_missing_debug_symbol)``.
 
+.. rst-class:: v1-9-0
+
+NCSDK-13949: TF-M Secure Image copies FICR to RAM on nrf9160
+  TF-M Secure Image copies the FICR to RAM address between 0x2003E000 and 0x2003F000 during boot on nrf9160.
+
+.. rst-class:: v1-9-0
+
+NCSDK-12306: Enabling debug configuration causes usage fault on nrf9160
+  When debug configuration :kconfig:`CONFIG_TFM_CMAKE_BUILD_TYPE_DEBUG` is enabled a usage fault is triggered during boot on nrf9160.
```

Comment:
```suggestion
.. rst-class:: v1-9-0

NCSDK-12306: Enabling debug configuration causes usage fault on nRF9160
  When the debug configuration :kconfig:`CONFIG_TFM_CMAKE_BUILD_TYPE_DEBUG` is enabled, a usage fault is triggered during boot on nRF9160.
```

Reply:
Is there a workaround we can provide to both these known issues, @Vge0rge ?

Reply:
For this one then not enabling CONFIG_TFM_CMAKE_BUILD_TYPE_DEBUG is the workaround.
Feels kind of redundant though. Should I still add it?